src/opnsense/mvc/app/models/OPNsense/Unbound/Unbound.xml

```diff
@@ -48,6 +48,22 @@
                     <yy>YoYo List</yy>
                 </OptionValues>
             </type>
+            <localzone type="OptionField">
+                <Required>N</Required>
+                <default>null</default>
+                <BlankDesc>Return null (0.0.0.0) A record (default)</BlankDesc>
+                <OptionValues>
+                    <always_nxdomain>Always return NXDOMAIN</always_nxdomain>
+                    <always_null>Always return null A/AAAA response (matches subdomains)</always_null>
+                    <redirect>Return NXDOMAIN or local data (matches subdomains)</redirect>
+                    <inform_redirect>Log and return NXDOMAIN or local data (matches subdomains)</inform_redirect>
+                    <inform>Log and resolve normally or return local data</inform>
+                </OptionValues>
+            </localzone>
+            <localdata type="BooleanField">
+                <default>0</default>
+                <Required>N</Required>
```

Comment:
Better set „Y“ here and bump model version for migration

Reply:
Will be updated in next push